server/main.go

```diff
@@ -30,19 +30,38 @@ func languagesHandler(w http.ResponseWriter, r *http.Request) {
 
 func runHandler(w http.ResponseWriter, r *http.Request) {
 	// TODO: parse request body
+	decoder := json.NewDecoder(r.Body)
+	var res api.RunRequest
+	err := decoder.Decode(&res)
 
+	if err != nil {
+		log.Printf("failed to parse request body: %b\n", err)
+	}
 	// TODO: transform request body into runner engine input
 
+	handler := coderunner.NewCodeRunner("api-runhandler", "")
+
+	RunProps := coderunner.RunnerProps{
+		Source: res.Source,
+		Lang:   res.Lang,
+	}
+
 	// TODO: let code runner run the code
 
+	RunnerOutput, err := handler.Run(&RunProps)
+
+	if err != nil {
+		log.Printf("failed to run output: %v\n", err)
+	}
+
 	// TODO: replace hard-coded reponse with transformed runner output
 	output := api.RunResponse{
-		Stdout: "hello world",
-		Stderr: "",
+		Stdout: RunnerOutput.Stdout,
+		Stderr: RunnerOutput.Stderr,
 		Error:  nil,
```

Comment:
Missed this before but this should return `RunnerOutput.CommandError` 
The type is from here: https://github.com/camerondurham/runner/blob/3ab31af13489868e0844b82ae19f8c3c1dc958a3/engine/coderunner/types.go#L32-L36